website/docs/tutorials/07-testing/03-fuzzing/reproducing-failures.md

```diff
@@ -6,14 +6,14 @@ As for now, Protostar provides one mean of achieving this.
 
 ## Reproducing a test run with seed
 
-When a test suite with fuzz test is run, Protostar will print out a numeric seed of a pseudo-random
-number generator used as a source of generated examples:
+Protostar prints a numeric seed of a pseudo-random number generator used as a source of generated
```

Comment:
Do we have a test checking if the seed changes with every run?

Reply:
#712